tools/flat-storage/src/commands.rs

```diff
@@ -329,6 +341,28 @@ impl FlatStorageCommand {
                     cmd.batch_size,
                 );
             }
+            SubCommand::ConstructTrieFromFlat(cmd) => {
+                let (_, epoch_manager, _, chain_store, store) = Self::get_db(
+                    &opener,
+                    home_dir,
+                    &near_config,
+                    near_store::Mode::ReadWriteExisting,
+                );
+
+                let write_opener = NodeStorage::opener(
+                    &cmd.write_store_path,
+                    false,
+                    &near_config.config.store,
+                    None,
+                );
+                let write_node_storage = write_opener.open_in_mode(Mode::Create).unwrap();
+                let write_store = write_node_storage.get_hot_store();
+
+                let tip = chain_store.final_head().unwrap();
```

Comment:
not that it matters here, but are we sure that chain's final_head is equal to the block that flat storage state is at? If not, maybe let's make a comment here so we don't mislead ourselves later.

Reply:
Added as part of --help and description of function